howso/utilities/internals.py

```diff
@@ -806,3 +806,23 @@ def to_pandas_datetime_format(f):
             # Failed to check pandas version
             pass
     return f
+
+
+class IgnoreFutureWarnings:
```

Comment:
Great job on this. What do you think about renaming to just `IgnoreWarnings` and allow the initializer to accept an Iterable of categories to ignore?

Reply:
Yea honestly I was really the exact same thing when writing this, but I didn't know if that was out of scope